redo/rules/build_style.py

```diff
@@ -87,14 +88,19 @@ def _build(self, redo_1, redo_2, redo_3):
 
         # Run codespell check:
         codespell_cwd = os.getcwd() + os.sep + "*.*"
+        codespell_ignore = expanduser("~") + os.sep + "adamant" + os.sep + "redo" + os.sep + "codespell" + os.sep + "ignore_list.txt"
```

Comment:
Use `os.environ["ADAMANT_DIR"]` instead of "~/adamant". This allows us to change the environment setup later and not have things break. Other build rules use this pattern, see build_clean.py for example.

Reply:
Much better, thanks, addressed in [b5cf61d](https://github.com/lasp/adamant/pull/11/commits/b5cf61d04d4181f943fd2726919e11847b195815)